google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerDialect.java

```diff
@@ -256,4 +259,48 @@ public SqmMultiTableInsertStrategy getFallbackSqmInsertStrategy(
       EntityMappingType entityDescriptor, RuntimeModelCreationContext runtimeModelCreationContext) {
     return NoOpSqmMultiTableInsertStrategy.INSTANCE;
   }
+
+  @Override
+  public String addSqlHintOrComment(
+      String sql,
+      QueryOptions queryOptions,
+      boolean commentsEnabled) {
+    if (hasCommentHint(queryOptions)) {
+      sql = applyHint(sql, queryOptions.getComment());
+    }
+    if (queryOptions.getDatabaseHints() != null && !queryOptions.getDatabaseHints().isEmpty()) {
```

Comment:
Cant we use CollectionUtils.isNotEmpty() from apache commons?

Reply:
Apache Commons is not a dependency in this library, so no.

Reply:
No, they're lists, so that won't work.

Reply:
Oops!